pom.xml

```diff
@@ -140,6 +140,11 @@
             <artifactId>lombok</artifactId>
         </dependency>
         <dependency>
+            <groupId>commons-codec</groupId>
+            <artifactId>commons-codec</artifactId>
+            <version>1.9</version>
```

Comment:
@ikhvostenkov there's a newer version (1.10) available. do you think we could use it?